dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -395,106 +416,133 @@ export const FeatureCard = ({
 								{/* This image overlay is styled when the CardLink is hovered */}
 								<div className="image-overlay" />
 
-								<div css={overlayStyles}>
-									{/**
-									 * Without the wrapping div the headline and
-									 * byline would have space inserted between
-									 * them due to being direct children of the
-									 * flex container
-									 */}
-									<div>
-										<CardHeadline
-											headlineText={headlineText}
+								<div css={contentStyles}>
+									{mainMedia?.type === 'Audio' &&
+										!!podcastImage?.src && (
+											<div
+												css={
+													podcastImageContainerStyles
+												}
+											>
+												<div css={podcastImageStyles}>
+													<CardPicture
+														mainImage={
+															podcastImage.src
+														}
+														imageSize="podcast"
+														alt={
+															podcastImage.altText ??
+															''
+														}
+														loading="lazy"
+														roundedCorners={true}
```

Comment:
It should only be app that use rounded corners for podcast images, web uses a square image https://www.figma.com/design/6NWdhzJfb3uraFgalGaNAE/Card-Components?node-id=4326-12608&m=dev

```suggestion
														roundedCorners={false}
```

Reply:
Nice catch, thanks!